pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1082,7 +1083,19 @@ public void checkGC(int gcIntervalInSeconds) {
         producers.forEach(Producer::checkPermissions);
         subscriptions.forEach((subName, sub) -> sub.getConsumers().forEach(Consumer::checkPermissions));
         checkMessageExpiry();
-        return checkReplication();
+        CompletableFuture<Void> result = new CompletableFuture<Void>();
+        checkReplication().thenAccept(res -> {
+            log.info("Policies updated successfully {}", data);
```

Comment:
We should include the topic name here. Also I don't think that `data` will print the json content and anyway it would be printing too much.